### ISSUE TITLE
Sparontologies does not content negotiate as expected. 

### ISSUE THREAD

**Issue by straleyb** [OregonDigital/OD2#376]
### Descriptive summary

In rails console, 
```
irb(main):022:0> graph = RDF::Graph.load('https://w3id.org/spar/mediatype/text/html')
=> #<RDF::Graph:0x2b1fb3ce3aa8(default)>
irb(main):023:0> graph.statements
=> []
```
doesnt work,

```
graph = RDF::Graph.load('http://id.loc.gov/authorities/subjects/sh85145447')
=> #<RDF::Graph:0x2b1fb3de87b4(default)>
irb(main):025:0> graph.statements
```
does. 

Something is still up with sparontologies that we cant get labels per expected.

**Comment by revgum** [comment]
Ensure that when fetching from spar, the url has a trailing `.rdf` so that it forces their side to not respond with HTTP 302 (or HTML) that isn't working for that ontology.

**Comment by straleyb** [comment]
This is fixed via #337
